third_party/protobuf/BUILD

```diff
@@ -0,0 +1,31 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # BSD
+
+genrule(
+    name = "protoc_bin",
+    srcs = select({
+        ":darwin": ["@protoc_macosx//file"],
+        "//conditions:default": ["@protoc_linux_x86_64//file"],
+    }),
+    outs = ["protoc"],
+    cmd = " && ".join([
+        "IN=$$(pwd)/$(SRCS)",
+        "OUT=$$(pwd)/$@",
+        "TMP=$$(mktemp -d $${TMPDIR:-/tmp}/genrule.XXXXXXXXXX)",
+        "cd $$TMP",
+    ]) + select({
+        ":darwin": " && unzip -q $$IN && ",
```

Comment:
1. Is the select necessary here, considering the two statements are identical?
2. Is it possible to pass a parameter to unzip that says, only extract this specific file?